src/main/java/de/hysky/skyblocker/skyblock/garden/LowerSensitivity.java

```diff
@@ -0,0 +1,45 @@
+package de.hysky.skyblocker.skyblock.garden;
+
+import de.hysky.skyblocker.config.SkyblockerConfigManager;
+import de.hysky.skyblocker.utils.ItemUtils;
+import de.hysky.skyblocker.utils.Location;
+import de.hysky.skyblocker.utils.Utils;
+import net.fabricmc.fabric.api.client.event.lifecycle.v1.ClientTickEvents;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.item.ItemStack;
+
+public class LowerSensitivity {
+
+    private static boolean sensitivityLowered = false;
+
+    public static void init() {
+        ClientTickEvents.END_WORLD_TICK.register(world -> {
+            if (!Utils.isOnSkyblock() || Utils.getLocation() != Location.GARDEN || MinecraftClient.getInstance().player == null) {
+                if (sensitivityLowered) lowerSensitivity(false);
+                return;
+            }
+            if (SkyblockerConfigManager.get().locations.garden.lockMouseTool) {
+                ItemStack mainHandStack = MinecraftClient.getInstance().player.getMainHandStack();
+                String itemId = ItemUtils.getItemId(mainHandStack);
+                boolean isHoldingTool = FarmingHudWidget.FARMING_TOOLS.containsKey(itemId);
+                if (isHoldingTool && !sensitivityLowered) lowerSensitivity(true);
+                else if (!isHoldingTool && sensitivityLowered) lowerSensitivity(false);
+
+            }
+            if (SkyblockerConfigManager.get().locations.garden.lockMouseGround) {
+                boolean onGround = MinecraftClient.getInstance().player.isOnGround();
+                if (onGround && !sensitivityLowered) lowerSensitivity(true);
+                else if (!onGround && sensitivityLowered) lowerSensitivity(false);
+            }
```

Comment:
I would make it so that the sensitivity is only lowered if the player is both holding a farming tool and on ground if on ground is turned on. Currently, it lowers the sensitivity when either of the two happens.

Reply:
Well maybe want either. But I can see and being wanted, not hard to do but idk how to put that in form in the config

Reply:
I don't think anyone would want either. It doesn't make sense to have your mouse locked if you just on ground and not holding a farming tool.

Reply:
I guess that makes sense. Should I just switch to an enum then? like OFF, HOLD_TOOL and HOLD_TOOL_ON_GROUND?

Reply:
The current system is fine just change `lockMouseGround` to `lockMouseGroundOnly` and the check to
```java
boolean isHoldingTool = FarmingHudWidget.FARMING_TOOLS.containsKey(itemId) && (!SkyblockerConfigManager.get().locations.garden. lockMouseGroundOnly || MinecraftClient.getInstance().player.isOnGround());
```